docs/whats-new/preview-3.md

```diff
@@ -0,0 +1,446 @@
+
+For existing .NET Aspire apps, after installing the latest workload, update all .NET Aspire package references to:
+
+- `8.0.0-preview.3.24105.21`
+
+For example, package references in your `.csproj` file for `Aspire.Hosting` should be updated to:
+
+```xml
+<PackageReference Include="Aspire.Hosting" Version="8.0.0-preview.3.24105.21" />
+```
+
+If you're using Visual Studio, use the NuGet Package Manager to update all packages for the solution. Ensure that the pre-release checkbox is checked.
+
+### API changes
+
+There are several hosting API changes in preview 3:
+
+- In existing code, `builder.WithServiceBinding(...)` should now be changed to `builder.WithEndpoint(...)` to retain the same behavior (there are similar changes for other resource types as well). The `WithServiceBinding` API has been obsoleted, while it will continue to work with a compile warning, you should update it before it's removed in the next preview.
+
+For more information on API changes, see [Breaking changes and new APIs](#breaking-changes-and-new-apis).
+
+## Dashboard updates and refactoring
+
+The [.NET Aspire Dashboard](../fundamentals/dashboard.md) has undergone a lot of refactoring, splitting it out into its own independent component. The dashboard used to be run as part of the host project, and that caused version conflicts between the dashboard and project dependencies. The dashboard is now its own independent executable, that communicates via gRPC to an endpoint to retrieve information about the projects, containers, their status and logs.
+
+This refactoring enables the dashboard to be more easily used for real-time runtime diagnostics after deployment, as hosts can implement the same gRPC contract/endpoint for project and dependency status.
+
+Refactoring shouldn't require any changes to .NET Aspire projects as there have been no related API changes for the projects themselves. Using the dashboard is unchanged in your development environment.
+
+### Localization and accessibility
+
+Many small changes across the dashboard have been made to support localization and accessibility. These changes include improving color contrast, adding accessibility attributes, and moving UI text into resource files.
+
+An example of an improvement is how the dashboard indicates trace length. Previously there was a subtle color gradient on each row that didn't meet accessibility requirements. In preview 3 this was replaced with a radial progress icon:
+
+:::image type="content" source="media/preview-3/trace-duration.png" lightbox="media/preview-3/trace-duration.png" alt-text="Trace details page with new radial icon used to quickly differentiate trace lengths.":::
+
+### New resource details view
+
+The resources page grid was bursting at the seams with information. In preview 3 there's now a resource details view. The grid continues to display important information, and additional details, including all of a resources properties, are available in the details view.
```

Comment:
```suggestion
The resources page grid was previously overloaded with information. Preview 3 now includes a resource details view for improved readability. The grid continues to display important information, while the details view shows additional data, including all of the resource's properties.
```

Just a personal opinion but this bit sounded a little odd to me - an alternate version here.